apiserver/facades/agent-schema.json

```diff
@@ -4751,14 +4751,6 @@
                         }
                     }
                 },
-                "SetObservedNetworkConfig": {
```

Comment:
This was a result of embedding. It was never called directly on this API anyway.